HOWTO.md

```diff
@@ -5,37 +5,35 @@ This How-To will explain how to write a basic **Matrix <--> Slack bridge** in un
 
 You need to have:
  - A working homeserver install
- - `npm` and `nodejs`
+ - `npm` and Node.js

```

Comment:
Personal preference, but it felt weird to have NPM come first. Also, having a mixture of brand and package names :)
```suggestion
 - Node.js (with NPM)
```